_posts/2024-05-23-get-involved-with-pyos.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title: "Get Involved with pyOpenSci"
+excerpt: "Fun fact: there are a multitude of ways for you to get involved with pyOpenSci! From social media to in-person events to joining our editoral team, this guide walks you through every opportunity available."
+author: "Jesse Mostipak"
+permalink: /blog/get-involved-with-pyos.html
+header:
+  overlay_image: images/blog/2024/may/23-05-2024-get-involved.png
+  overlay_filter: 0.4
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## <i class="fa-regular fa-heart"></i> The pyOpenSci mission
+Founded in 2018 by our Executive Director, [Leah Wasser](https://github.com/lwasser), pyOpenSci builds diverse community that supports free and open Python tools for processing scientific data. We also build technical skills needed to contribute to open source and that support open science. While a diverse, inclusive community is at our core, radiating out from it are the three petals–how we accomplish our community goals–of pyOpenSci. These are:
```

Comment:
I love this. i've been playing with a bit more powerful language around pyOpenSci supports scientists in tackling the world's greatest challenges by helping them make their science more open and collaborative. ... and then we do this by ... building diverse community and technical skills...